### DISCUSSION TITLE
Resource usage of the Pequeno Core in Artix-7

### DISCUSSION THREAD

**Discussion by sckid1108** [iammituraj/pequeno_riscv#3] [Q&A]
What was the resource utilization of this core ?

**Comment by iammituraj** [comment]
**Synthesis summary on FPGA**
----------------------------------
[UPDATED: 25-Aug-2025]
**Target** - Artix-7, xc7a35tcpg236-1
**LUTs** - 1076
**Registers** - 552
**Targetted clock freq** - 100 MHz
**Core version** - v1.0.2, Configuration - Light

**Comment by sckid1108** [comment]
Excellent thank you !

On Nov 28, 2024 10:36 AM, Mitu Raj ***@***.***> wrote:

Synthesis summary on FPGA

Target - Artix-7, xc7a35tcpg236-1
LUTs - 2084
Registers - 1564
Targetted clock freq - 100 MHz
